test/e2e/backup_test.go

```diff
@@ -0,0 +1,63 @@
+package e2e
+
+import (
+	"flag"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/vmware-tanzu/velero/pkg/cmd/cli/install"
+)
+
+var (
+	uuidgen              uuid.UUID
+	err                  error
```

Comment:
Does this `err` var and the one on line 42 need to be in outer scopes from where they are used and checked?

Reply:
the error vars in outter scopes tend to be ignored when adding more tests. To cater to that, I will make them local to the scope they are used in.